### PR TITLE
Summary of Mind and Matter

### PR DESCRIPTION
**Pull request by JJM317**

### CODE REVIEW

File: _content/articles/summary-of-mind-and-matter_saddhatissa-hammalawa.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: "The Summary of Mind and Matter"
+authors:
+  - "Hammalawa Saddhatisa"
+external_url: "https://palitextsociety.org/wp-content/uploads/2022/01/JPTS_1987_XI.pdf"
```
Comment (khemarato) [inline _content/articles/summary-of-mind-and-matter_saddhatissa-hammalawa.md:5]: That's a fine source_url. Here's the proper external_url: https://archive.org/download/jpts-xi-1987/Namarupasamaso_%20Summary%20of%20Mind%20and%20Matter%20-%20H%20Saddhatissa_text.pdf

File: _content/articles/summary-of-mind-and-matter_saddhatissa-hammalawa.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+drive_links:
+  - "https://drive.google.com/file/d/1Q8ThAi8KBgCrPFJjgVi6CfU3zcYPLn47/view?usp=sharing"
+course: abhidhamma
+tags:
+  - inner
```
Comment (khemarato) [inline _content/articles/summary-of-mind-and-matter_saddhatissa-hammalawa.md:10]: I think just `consciousness` for a tag? 🤔

**Comment by JJM317** [comment]
I cannot find PR #366 in VS.  When I click on the branch, no PR is
associated with it. Did you make the fixes and merge? Did I do something
goofy again?

[image: image.png]
[image: image.png]

**Comment by khemarato** [comment]
Yeah, I just went ahead and made the fixes myself 😊  Sorry for the confusion!

**Pull request merged**